src/plugins/workspace/server/saved_objects/workspace_ui_settings_client_wrapper.ts

```diff
@@ -111,7 +127,9 @@ export class WorkspaceUiSettingsClientWrapper {
           options
         );
 
-        configObject.attributes = workspaceUpdateResult.attributes.uiSettings;
+        if (workspaceUpdateResult.attributes.uiSettings) {
```

Comment:
Is it necessary to check if `workspaceUpdateResult.attributes.uiSettings` is null? Because for `update` case, the uiSettings seems cannot be `null`.

Reply:
There is a type error saying that configObject.attributes can not be null so that I added a `if`, type assertion should work as well but I feel it is more safe to check.

Reply:
I see, that makes sense